MonkeyChat/ChatViewController.swift

```diff
@@ -554,6 +543,8 @@ extension ChatViewController {
      */
     
     self.send(text ?? "", size: self.maxSize)
+    NotificationCenter.default.post(name: Notification.Name(rawValue: MonkeyChatNotification.messageSent), object: self)
```

Comment:
This would change too with the proposed change to the struct to something like
`NotificationCenter.default.post(name: Notification.Name.MonkeyChatNotification.messageSent, object: self)`